File: frontend/kasicuts_app/lib/models/barber_work.dart
class BarberWork {
  final int id;
  final int barberId;
  final String barberName;
  final String styleName;
  final String description;
  final String imageUrl;
  final double priceFrom;

  BarberWork({
    required this.id,
    required this.barberId,
    required this.barberName,
    required this.styleName,
    required this.description,
    required this.imageUrl,
    required this.priceFrom,
  });

  factory BarberWork.fromJson(Map<String, dynamic> json) {
    return BarberWork(
      id: json['id'],
      barberId: json['barberId'],
      barberName: json['barberName'],
      styleName: json['styleName'],
      description: json['description'],
      imageUrl: json['imageUrl'],
      priceFrom: (json['priceFrom'] as num).toDouble(),
    );
  }
}

File: backend/KasiCuts.Api/Models/BarberWork.cs
namespace KasiCuts.Api.Models
{
    public class BarberWork
    {
        public int Id { get; set; }

        public int BarberId { get; set; }

        public string BarberName { get; set; } = string.Empty;

        public string StyleName { get; set; } = string.Empty;

        public string Description { get; set; } = string.Empty;

        public string ImageUrl { get; set; } = string.Empty;

        public decimal PriceFrom { get; set; }

        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    }
}